src/frontends/pytorch/src/op/flatten.cpp

```diff
@@ -3,59 +3,65 @@
 //
 
 #include "openvino/frontend/pytorch/node_context.hpp"
-#include "openvino/opsets/opset10.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/slice.hpp"
+#include "openvino/op/squeeze.hpp"
+#include "openvino/op/unsqueeze.hpp"
 
 namespace ov {
 namespace frontend {
 namespace pytorch {
 namespace op {
 
+using namespace ov::op;
+
 OutputVector translate_flatten(NodeContext& context) {
     auto start_dim = context.const_input<int64_t>(1);
     auto end_dim = context.const_input<int64_t>(2);
 
-    auto shape = std::make_shared<opset10::ShapeOf>(context.get_input(0), element::i32);
-    auto rank_ = std::make_shared<opset10::ShapeOf>(shape, element::i32);
-    auto rank = std::make_shared<opset10::Squeeze>(rank_);
-    // Use opset::If for dim normalization
+    auto shape = std::make_shared<v3::ShapeOf>(context.get_input(0), element::i32);
+    auto rank_ = std::make_shared<v3::ShapeOf>(shape, element::i32);
+    auto rank = std::make_shared<v0::Squeeze>(rank_);
```

Comment:
```suggestion
    Output<Node> rank = std::make_shared<v3::ShapeOf>(shape, element::i32);
    rank = std::make_shared<v0::Squeeze>(rank);
```

Reply:
Also, it is good to have such primitive in utils to compute a rank. Please consider - very useful stuff:)

Reply:
Agree, will do it separately